frontend/src/components/settings/ExpandDropdown.vue

```
@@ -0,0 +1,597 @@
+  emits: ["update:modelValue"],
+
+  data() {
+    return {
+      open: false,
+      panelOpen: false,
+      isExpanded: false,
+      closing: false,
+      searchQuery: "",
+      overlayStyle: {},
+      shadowStyle: {},
+      localInputId: `expand-dropdown-${expandDropdownIdCounter += 1}`,
```

Biome (2.5.0)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)